data/multiplayer/scenarios/ANL_utils/ANL_research_options.cfg

```diff
@@ -154,9 +154,10 @@ Our mines produce $player_$side_number|.mining.gold|g
                         # Farming
 
                         [option]
-                            message= {MENU_IMG_TXT "items/flower4.png" ( _ "<span color='green'>Agriculture</span>
-Farmers produce +1 gold
-Study Progress: $player_$side_number|.farming.progress|/$player_$side_number|.farming.target")}
+                            image = "items/flower4.png"
+                            message = _ "<span color='green'>Agriculture</span>
+    Farmers produce +1 gold
+    Study Progress: $player_$side_number|.farming.progress|/$player_$side_number|.farming.target"
                             [command]
```

Comment:
Hmm... I don't recall putting that there, or anywhere, probably it was wmlindent, or an accidental whitespace when I did not notice...